### PR TITLE
Avalanche (AVAX)

### PR DESCRIPTION
**Pull request by EvanJRichard**

### CODE REVIEW

**Comment by catenocrypt** [comment]
Note: This adds logo and basic info about new Avalanche network, but this in itself will not mean full support.
Submission looks OK so far, awaiting fee payment.

File: blockchains/avalanche/info/info.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "explorer": "https://explorer.avax.network",
+    "research": "https://research.binance.com/en/projects/avalanche",
+    "symbol": "AVAX",
+    "type": "coin",
+    "decimals": 9,
+    "status": "active",
+    "tags": [
+        "staking-native"
+    ]
+}
```
Comment (vikmeup) [inline blockchains/avalanche/info/info.json:16]: For the logo can you use just a symbol, similar to CMC: https://coinmarketcap.com/currencies/avalanche/

Comment (EvanJRichard) [reply blockchains/avalanche/info/info.json:16]: Can do! Logo updated.

**Comment by catenocrypt** [comment]
Fine, merging.

**Comment by catenocrypt** [comment]
@EvanJRichard , please duplicate to 2 chains. 
In info.json files differentiate name, in the description mention chain, and check decimals.
Logo can be same, or better with some variation (e.g. different color, or other visual clue).

**Comment by EvanJRichard** [comment]
@catenocrypt Thanks! https://github.com/trustwallet/assets/pull/6134

**Pull request merged**